app/javascript/packs/access_tokens.scss

```diff
@@ -0,0 +1,2 @@
+@import '~@patternfly/patternfly/components/DescriptionList/description-list.css';
```

Comment:
How come all these missing imports were not needed before?

Reply:
Yes, correct. Before `provider.scss` added *all* stylesheets, such as base, individual components and quickstarts. But this is bad for 3 reasons:
- There is no code splitting, all stylesheets are always loaded together.
- It's harder to track what stylesheets are necessary.
- It fosters duplicated (or even triplicated) stylesheets.

Keep in mind that Patternfly components rendered by React include their stylesheets, therefore we only need to import css like this when we use HTML classes in our ruby templates, e.g.: `<button class="pf-c-button">OK</button>`. Before, a page featuring a Card would load both:
- `node_modules/patternfly/components/Card/card.css` (from provider.scss)
- `node_modules/patternfly/react-styles/css/components/Card/card.css` (from Card.tsx)

Reply:
Thanks for the explanation, that's a great improvement.

I'm just wondering if it might be better to do it in a different PR. If it's refactoring, not affecting the behavior or the looks, I'd separate it from webpacker removal.

Reply:
In all honestly, at this point I would not know how to separate it 😅